### ISSUE TITLE
Python path contain Chinese character in red.xml throw exception

### ISSUE THREAD

**Issue by kenifanying** [nokia/RED#316]
I have a simple robot project, and config python path that contain Chinese character in red.xml. The python library can be automatically discovered by RED but marked non-accessible. I tried to reload the library get some error:
```
Library specification generation problem

Problem occurred during generating libdoc for 'Telnet' library located at 'C:\Users\tony\robotframework\robot_test\lib\Telnet.py'.
RED python session problem. Following exception has been thrown by python service:
            Traceback (most recent call last):
              File "C:\Users\tony\AppData\Local\Temp\RobotTempDir8198076229433796343\robot_session_server.py", line 42, in inner
                result['result'] = func(*args, **kwargs)
              File "C:\Users\tony\AppData\Local\Temp\RobotTempDir8198076229433796343\robot_session_server.py", line 91, in inner
                return to_call(*args, **kwargs)
              File "C:\Users\tony\AppData\Local\Temp\RobotTempDir8198076229433796343\robot_session_server.py", line 124, in inner
                return to_call(*args, **kwargs)
              File "C:\Users\tony\AppData\Local\Temp\RobotTempDir8198076229433796343\robot_session_server.py", line 64, in inner
                return func(*args, **kwargs)
              File "C:\Users\tony\AppData\Local\Temp\RobotTempDir8198076229433796343\robot_session_server.py", line 370, in create_libdoc_in_separate_process
                raise Exception(err.decode('UTF-8', 'replace'))
            Exception: Traceback (most recent call last):
              File "C:\Users\tony\AppData\Local\Temp\RobotTempDir8198076229433796343\red_libraries.py", line 122, in <module>
                python_paths = __decode_unicode_if_needed(sys.argv[3].split(";"))
              File "C:\Users\tony\AppData\Local\Temp\RobotTempDir8198076229433796343\red_libraries.py", line 111, in __decode_unicode_if_needed
                return [__decode_unicode_if_needed(elem) for elem in arg]
              File "C:\Users\tony\AppData\Local\Temp\RobotTempDir8198076229433796343\red_libraries.py", line 109, in __decode_unicode_if_needed
                return arg.decode('utf-8')
              File "C:\Python27\lib\encodings\utf_8.py", line 16, in decode
                return codecs.utf_8_decode(input, errors, True)
            UnicodeDecodeError: 'utf8' codec can't decode bytes in position 49-50: invalid continuation byte
            
```

This issuse may be related to #315, I upload the demo project 'robot_test`, any assistance would be appreciated.

Python 2.7.16
Robot Framework 3.1.1 (Python 2.7.16 on win32)
RED - Robot Editor version 0.8.12
System: Windows 7 SP1 X86_64

[robot_test.zip](https://github.com/nokia/RED/files/3328535/robot_test.zip)

**Comment by michalanglart** [comment]
I'm unable to reproduce this problem on both RED 0.8.12 and 0.8.13 and under Python 2.7.15 or 3.7.1

In my case the library is discovered, added to red.xml and specification file properly generated. Library reloading works manually as well as automatically when the `Telnet.py` is changed.

**Comment by kenifanying** [comment]
> 
> 
> I'm unable to reproduce this problem on both RED 0.8.12 and 0.8.13 and under Python 2.7.15 or 3.7.1
> 
> In my case the library is discovered, added to red.xml and specification file properly generated. Library reloading works manually as well as automatically when the `Telnet.py` is changed.

Please try unzip it directly and add the unzip directory robot_test by using "Open Projects from File System" menu.

**Comment by modrz** [comment]
I was managed to reproduce your issues. Both #315 and #316 should be fixed in next version.

**Comment by michalanglart** [comment]
Fixed in 0.9.0
